ansible/group_vars/dockermanager-hosts.yml

```diff
@@ -1,6 +1,6 @@
 docker_use_ipv4_nic_mtu: True
 
 #idr_jupyter_hub_image: "imagedata/jupyterhub-githubauth:0.3.0"
-idr_jupyter_notebook_image: "imagedata/jupyter-docker:0.3.0"
+idr_jupyter_notebook_image: "imagedata/jupyter-docker:develop"
```

Comment:
I've bumped the version, you'll need to rerun travis after you push the image tag.